From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: phuzion <398094+phuzion@users.noreply.github.com>
Date: Wed, 19 May 2021 18:05:36 +0000
Subject: [PATCH] This commit changes the official Quassel IRC channel,
 (#quassel) from the Freenode IRC network to the Libera IRC network in
 response to recent issues which resulted in the entire volunteer staff of
 Freenode stepping down.

For more information about this situation, please read this:
https://gist.github.com/joepie91/df80d8d36cd9d1bde46ba018af497409
---
 README.md             | 6 ++----
 data/networks.ini     | 7 +++++--
 src/qtui/aboutdlg.cpp | 2 +-
 3 files changed, 8 insertions(+), 7 deletions(-)

diff --git a/README.md b/README.md
index c7b03dc2..6f54dfaa 100644
--- a/README.md
+++ b/README.md
@@ -47,31 +47,29 @@ documentation.
 
 IRC is the preferred means of getting in touch with the developers.
 
-The Quassel IRC Team can be contacted on **`Freenode/#quassel`**
-(or **`#quassel.de`**).  If you have trouble getting Quassel to connect,
-you can use [Freenode's webchat][help-freenode].
+The Quassel IRC Team can be contacted on **`Libera Chat/#quassel`**
+(or **`#quassel.de`**). 
 
 We always welcome new users in our channels!
 
 You can learn more and reach out to us in several ways:
 * [Visit our homepage][web-home]
 * [Submit and browse issues on the bugtracker][dev-bugs]
   * GitHub issues are not used, but [pull requests][dev-pr-new] are accepted!
 * [Email the dev team - devel@quassel-irc.org][dev-email]
 
 Thanks for reading,
 
 ~ *The Quassel IRC Team*
 
 [web-home]: https://quassel-irc.org
 [web-download]: https://quassel-irc.org/downloads
 [dev-bugs]: https://bugs.quassel-irc.org
 [dev-email]: mailto:devel@quassel-irc.org
 [dev-pr-new]: https://github.com/quassel/quassel/pull/new/master
 [docs-wiki]: https://bugs.quassel-irc.org/projects/quassel-irc/wiki
 [docs-wiki-unofficial-build]: https://bugs.quassel-irc.org/projects/quassel-irc/wiki#Unofficial-builds
 [docs-wiki-getstart]: https://bugs.quassel-irc.org/projects/quassel-irc/wiki#Getting-started
-[help-freenode]: https://webchat.freenode.net?channels=%23quassel
 [repo-changelog]: ChangeLog
 [ci-linux-badge]: https://travis-ci.org/quassel/quassel.svg?branch=master
 [ci-linux-status-page]: https://travis-ci.org/quassel/quassel/branches
diff --git a/data/networks.ini b/data/networks.ini
index 73afecb3..cf16313e 100644
--- a/data/networks.ini
+++ b/data/networks.ini
@@ -19,22 +19,25 @@ Servers=irc.efnet.info:6667,irc.efnet.org:6667,irc.inter.net.il:6667,irc.igs.ca:
 Servers=irc.enterthegame.com:6667
 
 [Freenode]
-Default=Yes
-DefaultChannels=#quassel
 Servers=chat.freenode.net:+6697,chat.freenode.net:6667
 
 [GalaxyNet]
 Servers=irc.galaxynet.org:6662,irc.galaxynet.org:7000,boston.ma.us.galaxynet.org:6661
 
 [GameSurge]
 Servers=irc.gamesurge.net:6667,irc.eu.gamesurge.net:6667
 
 [GamesNET]
 Servers=irc.gamesnet.net:6667
 
 [IRCnet]
 Servers=irc.ircnet.com:6667,open.ircnet.net:6666,irc.us.ircnet.net:6665,ircnet.netvision.net.il:6666,irc.tokyo.wide.ad.jp:6666,irc.freebsd.org.tw:6666,linz.irc.at:6666,vienna.irc.at:6666,ircnet.realroot.be:6666,irc.datacomm.ch:6664,irc.felk.cvut.cz:6667,random.ircd.de:6666,irc.dotsrc.org:6666,irc.ircnet.ee:6666,irc.cs.hut.fi:6667,ircnet.club-internet.fr:6666,atw.irc.hu:6667,irc.simnet.is:6660,irc.eutelia.it:6664,irc.eutelia.it:7000,irc.tin.it:6665,irc.apollo.lv:6666,irc.apollo.lv:7000,irc.uunet.nl:6660,irc.xs4all.nl:6660,irc.snt.utwente.nl:6660,irc.ifi.uio.no:6667,irc.pvv.ntnu.no:6667,lublin.irc.pl:6666,warszawa.irc.pl:6666,irc.ludd.luth.se:6661,irc.swipnet.se:6660,irc.swipnet.se:8000,irc.arnes.si:6666,irc.link.si:6666,irc.nextra.sk:6666,ircnet.demon.co.uk:6665,ircnet.eversible.com:6665,ircnet.choopa.net:6665
 
+[Libera Chat]
+Default=Yes
+DefaultChannels=#quassel
+Servers=irc.libera.chat:+6697,irc.libera.chat:6667
+
 [OFTC]
 Servers=irc.oftc.net:+6697,irc.oftc.net:6667
 
diff --git a/src/qtui/aboutdlg.cpp b/src/qtui/aboutdlg.cpp
index 6b21b0f5..a6c4da25 100644
--- a/src/qtui/aboutdlg.cpp
+++ b/src/qtui/aboutdlg.cpp
@@ -67,7 +67,7 @@ QString AboutDlg::about() const
     QString res {tr("<b>A modern, distributed IRC Client</b><br><br>"
              "&copy;%1 by the Quassel Project<br>"
              "<a href=\"https://quassel-irc.org\">https://quassel-irc.org</a><br>"
-             "<a href=\"irc://irc.freenode.net/quassel\">#quassel</a> on <a href=\"https://www.freenode.net\">Freenode</a><br><br>"
+             "<a href=\"irc://irc.libera.chat/quassel\">#quassel</a> on <a href=\"https://libera.chat\">Libera Chat</a><br><br>"
              "Quassel IRC is dual-licensed under <a href=\"https://www.gnu.org/licenses/gpl-2.0.txt\">GPLv2</a> and "
                  "<a href=\"https://www.gnu.org/licenses/gpl-3.0.txt\">GPLv3</a>.<br>"
              "<a href=\"https://api.kde.org/frameworks/breeze-icons/html\">Breeze icon theme</a> &copy; Uri Herrera and others, licensed under the "
